src/ape/_cli.py

```diff
@@ -36,14 +46,17 @@ def invoke(self, ctx) -> Any:
             return super().invoke(ctx)
         except click.UsageError as err:
             self._suggest_cmd(err)
-        except ApeException as err:
-            if logger.level == LogLevel.DEBUG.value:
-                tb = traceback.format_exc()
-                err_message = tb or str(err)
-            else:
-                err_message = str(err)
+        except TransactionError as err:
+            if not err.traceback:
+                raise abort(err) from err
+
+            reporter = ctx.obj.chain_manager._reports
```

Comment:
- [ ] Either use the `__traceback__` that gets set now, or manually include the frame(s) from the script or plugin making the call that causes the tx err